operating/view-dashboard.adoc

```diff
@@ -6,58 +7,100 @@ include::modules/common-attributes.adoc[]
 //TODO: Add link to policy categories
 
-[id="system-violations"]
-== System Violations
+[id="overview-data"]
 
-The *System Violations* widget (upper-left) shows the number of critical, high, medium, and low severity violations on the *Dashboard* view.
-You can select a number on the widget to view details about those violations.
+The top of the dashboard shows an overview of system data, including the number of clusters, nodes, violations, deployments, images, and secrets. Choosing one of those items takes you to the main menu page for the item. For example, select *Deployments* to view the *Configuration Management* -> *Deployments* page.
 
-[id="compliance"]
-== Compliance
+[id="select-resources"]
+== Selecting resources for data display
 
-The *Compliance* widget (upper-right) shows the status of various compliance benchmarks and if their controls are passing or failing.
+You can review metrics for your entire system or only selected resources by selecting the following options:
 
-[NOTE]
-====
-The *Compliance* widget show details only after you run a compliance scan.
-====
-//TODO: Add link to compliance scan
+* To view data for specific clusters, select the clusters from the *Resources* drop-down list. 
+* To view data for all clusters, ensure *All clusters* is selected. 
+* To view data for specific namespaces, select them from the *Resources* drop-down list. Note that all namespaces are automatically included when you select *All clusters*. 
+
+[id="policy-violations-by-severity"]
+== Policy violations by severity
+
+The *Policy violations by severity* widget displays the total number of violations and displays the number of violations by severity level. Severity levels are color-coded with gray for low severity; yellow for medium severity; orange for high severity; and red for critical severity. 
+
+You can select items to view additional information as follows:
+
+* To view all violations of a specific severity, click the count for the desired severity level. 
+* To view all violations of any severity, select *View all*. Choosing a violations category or *View all* displays the *Violations* page and the menu bar reflects any *Cluster* or *Namespace* selections that you made in the dashboard.
+
+The *Policy violations by severity* widget also displays the three most recent violations that have a severity level of *Critical*. Choosing a specific violation displays the xref:../operating/respond-to-violations.adoc#view-violation-details[Violations detail] page with the clusters and namespaces you selected for display in the dashboard.
+
+[id="images-at-most-risk"]
+== Images at most risk
+
+The *Images at most risk* widget provides a list of images that have been determined to have the highest risk. ##Risk is determined by an analysis that weighs the risk priority, number of critical common vulnerabilities and exposures (CVEs), and number of important CVEs.##
```

Comment:
I'm not entirely sure what goes into the Risk score. I believe applied network policies have an impact as well but I couldn't describe how. I think Oscar Ward on the core workflows team is the person who could give you more info on how this score is determined.